tgstation.dme

```diff
@@ -103,6 +103,7 @@
 #include "code\_onclick\hud\alien.dm"
 #include "code\_onclick\hud\alien_larva.dm"
 #include "code\_onclick\hud\blob_overmind.dm"
+#include "code\_onclick\hud\blobbernauthud.dm"

```

Comment:
I wish the hud creation procs weren't in these so that unticking one accidentally would fucking THROW AN ERROR
I'm not sure when this happened but goddamn.